src/csharp/Grpc.Core/build/net45/Grpc.Core.targets

```diff
@@ -10,22 +10,22 @@
     the Grpc.Core.dll assembly and the native libraries are used at all times.
   -->
   <ItemGroup Condition="'$(Grpc_SkipNativeLibsCopy)' != 'true'">
-    <Content Include="$(MSBuildThisFileDirectory)..\..\runtimes\win\native\grpc_csharp_ext.x86.dll">
+    <Content Include="$(MSBuildThisFileDirectory)..\..\runtimes\win-x86\native\grpc_csharp_ext.dll">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Link>grpc_csharp_ext.x86.dll</Link>
```

Comment:
wouldn't the <Link> need to be updated as well?

Reply:
the problem is that the NativeExtension would need to be able to find the file that's copied to the output folder  on net45 (but you just changed its logic to look for grpc_csharp_ext.dll only.

At the same time, there are 2 `<Content>` items: one for x86 and one for x64.
you cannot copy them both into a single grpc_csharp_ext.dll (I think this is the original reason why we have the x64.dll and x86.dll suffix - to avoid a name clash in net45.